rest/utilities_testing.go

```diff
@@ -813,6 +813,7 @@ func EnableBlipSyncLogs() {
 	base.EnableLogKey("HTTP+")
 	base.EnableLogKey("Sync")
 	base.EnableLogKey("Sync+")
-	base.EnableLogKey("Sync++")
-
+	base.EnableLogKey("SyncMsg")
+	base.EnableLogKey("SyncMsg+")
+	base.EnableLogKey("WS")
```

Comment:
Might SyncMsg, SyncMsg+ and WS be too verbose to enable on every blip test?

Reply:
Ok I removed them.  I guess they can be added when needed for debugging.